test/persistence-hooks.suite.js

```diff
@@ -119,7 +136,8 @@ module.exports = function(dataSource, should, connectorCapabilities) {
           });
       });
 
-      it('triggers correct hooks when near filter is used', function(done) {
+      it('triggers the loaded hook multiple times when multiple instances exist with near filter',
```

Comment:
"when near filter is given" or "when near filter is used" to match the terms used in other test cases